benchmarks/benchmarks/linprog_benchmark_files/summary.txt

```diff
@@ -0,0 +1,102 @@
+*	Original SIF (MPS) files from http://www.cuter.rl.ac.uk/Problems/netlib.shtml
```

Comment:
Would be useful to state that this file is not used for benchmarking, it's only documentation. I was looking for the place where ``summary.txt`` was being read in to look at optimal values.